### DISCUSSION TITLE
Minimum version of postgres supported?

### DISCUSSION THREAD

**Discussion by jfreeman-hbo** [Apicurio/apicurio-registry#2543] [Q&A] [answered]
Hi,

What is the minimum version of postgreSql supported by apicurio registry? I see v12 mentioned in the documentation, but wondering if v11.9, or even v10 might work just fine?

**Comment by EricWittmann** [comment, marked as answer]
We haven't tested anything prior to 12.  When you say what is "supported", are you referring to official Red Hat support or are you asking what will actually work?  :)  Since you're asking here (in the open source project) I'm assuming it's the latter.  In that case, I'm **guessing** that PG 9 and later will work.  We're not doing anything particularly clever in the DB layer.

So give it a try and report back!  :)  We're happy to help work through issues when possible.

**Reply by jfreeman-hbo** [reply]
Thanks, that makes sense. Verified it works just fine in v10 and v11.
